drivers/sensors/ms56xx_uorb.c

```diff
@@ -286,7 +286,7 @@ static inline void baro_measure_read(FAR struct ms56xx_dev_s *priv,
 
   /* Wait data acquisition */
 
-  up_udelay(10000);
+  nxsig_usleep(10000);
```

Comment:
Why not replace all up_udelays() to nxsig_usleep()(? I think this will be more efficient in CPU utilization.